### PR TITLE
21931 submit staff review

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#21931

*Description of changes:*
  - included a date fix in business-pay for review creation

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

File: legal-api/src/legal_api/resources/v2/admin/reviews.py [hunk 1]
```diff
@@ -35,7 +38,69 @@ def get_reviews():
...
+        if review.status not in [ReviewStatus.AWAITING_REVIEW, ReviewStatus.RESUBMITTED]:
+            return jsonify({'message': 'No changes allowed.'}), HTTPStatus.BAD_REQUEST
+    else:
+        return jsonify({'message': 'Review not found.'}), HTTPStatus.NOT_FOUND
+
+    json_input = request.get_json()
+    status = json_input.get('status')
+    comment = json_input.get('comment')
+    if not status or not comment:
+        return jsonify({'message': 'Status and Comment are required.'}), HTTPStatus.BAD_REQUEST
```
Comment (severinbeauvais) [inline legal-api/src/legal_api/resources/v2/admin/reviews.py:62]: I believe comment is optional when the status is Approved.

Should this be "not status and not comment" for ChangeRequested and Rejected?

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/resources/v2/admin/reviews.py:62]: oh! I thought there will be a default comment from UI

Comment (severinbeauvais) [reply legal-api/src/legal_api/resources/v2/admin/reviews.py:62]: This is what the approved payload looks like:

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/resources/v2/admin/reviews.py:62]: * if status or comment is missing

Comment (vysakh-menon-aot) [reply legal-api/src/legal_api/resources/v2/admin/reviews.py:62]: > This is what the Approved payload looks like:
> 
> image

will change it

**Pull request merged**